charts/terrarium/values.yaml

```diff
@@ -5,51 +5,59 @@ modules_api_v1_service:
   replicas: 1
   image:
     repository: ghcr.io/terrariumcloud/terrarium
-    tag: v0.0.74
+    tag: v0.0.76-CIEDEV-3005.2
+    pullPolicy: Always
+providers_api_v1_service:
+  ingress:
+    path: /providers/v1
+  replicas: 1
+  image:
+    repository: ghcr.io/terrariumcloud/terrarium
+    tag: v0.0.76-CIEDEV-3005.2
```

Comment:
Ensure that you specify the correct version once the PR for terrarium has been merged and a new version becomes available, since this was a test release. Otherwise, looks good.

Reply:
Yes, sure
Updated the latest released version after the  merge of terrarium with support for provider registry protocol implementation.